contributing_to_docs/doc_guidelines.adoc

```diff
@@ -268,6 +271,84 @@ another topic, see the following usage:
 
 http://asciidoctor.org/docs/user-manual/#include-multiple
 
+== Writing concepts
+Concepts should support the things that users want to do and should not include
+task information like commands or numbered steps.
+
+== Writing tasks
+Write tasks in the following format.
+
+Task Title: Use a verb in the task title (for example, Create or Creating).
+
+Include a paragraph explaining why the user must perform this task. This should be 1-2 sentences maximum.
+
+.About This Procedure
+
+Include any gotchas (things that could trip up the user or cause the task to fail).
```

Comment:
"About This Procedure" does seem a bit broad; it's not immediately obvious from the title that the section is going to be about gotchas. Maybe "Precautions"... "Procedure Tips"... "Considerations"... ?